nerfstudio/data/dataparsers/phototourism_dataparser.py

```diff
@@ -176,6 +191,73 @@ def _generate_dataparser_outputs(self, split="train"):
             scene_box=scene_box,
             dataparser_scale=scale_factor,
             dataparser_transform=transform_matrix,
+            metadata=metadata,
         )
 
         return dataparser_outputs
+
+    def _load_3D_points(self, colmap_path: Path, transform_matrix: torch.Tensor, scale_factor: float):
```

Comment:
are there any differences between `_load_3D_points` here and the version in the colmap dataparser? or is it duplicated?

Reply:
Yes this part was copied from the colmap dataparser